examples/ars.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+// Ars Technica, Open Source tag example
+
+const Libingester = require('libingester');
+const thenifyAll = require('thenify-all');
+const RssToJson = thenifyAll(require('rss-to-json'), {}, ['load']);
+const fs = thenifyAll(require('fs'), {}, ['writeFile']);
+
+function remove_intermediate($, selector) {
+    $(selector).each((i, elem) => $(elem).replaceWith($(elem).contents()));
+}
+
+const COMMENT_NODE = 8;
+
+function ingest_article(hatch, {title, description, url, created}) {
+    return Libingester.util.fetch_html(url)
+    .then($ => {
+        const BASE_URI = Libingester.util.get_doc_base_uri($, url);
+        let asset = new Libingester.NewsArticle();
+
+        console.log('processing', title);
+        asset.set_title(title);
+        asset.set_synopsis(description);
+        asset.set_date_published(created);
+        asset.set_last_modified_date(new Date(created));
+        asset.set_source_name('Ars Technica');
+        asset.set_license('Proprietary');
+        asset.set_section('open-source');
+
+        let canonical_uri = $('head link[rel="canonical"]').attr('href');
+        asset.set_canonical_uri(canonical_uri);
+        asset.set_read_more_link(`
+<p>
+    Read more at <a href="${canonical_uri}">Ars Technica</a>
+</p>
+        `);
+
+        let thumb_uri = $('meta[property="og:image"]').attr('content');
+        let thumb_asset = Libingester.util.download_image(thumb_uri);
+        hatch.save_asset(thumb_asset);
+        asset.set_thumbnail(thumb_asset);
+
+        // Clean up
+        $('div').filter(function () {
```

Comment:
No, that binds a value of `this` that overrides the value that cheerio binds, so `$(this)` wouldn't work inside.